FFMpegCore/FFMpeg/FFMpeg.cs

```diff
@@ -60,11 +62,12 @@ public static async Task<bool> SnapshotAsync(string input, string output, Size?
         /// <param name="input">Source video file.</param>
         /// <param name="captureTime">Seek position where the thumbnail should be taken.</param>
         /// <param name="size">Thumbnail size. If width or height equal 0, the other will be computed automatically.</param>
+        /// <param name="streamIndex">Selected video stream index.</param>
         /// <returns>Bitmap with the requested snapshot.</returns>
-        public static Bitmap Snapshot(string input, Size? size = null, TimeSpan? captureTime = null)
+        public static Bitmap Snapshot(string input, Size? size = null, TimeSpan? captureTime = null, int streamIndex = 0)
```

Comment:
I think making `streamIndex` nullable and providing `null` as the default value would be better, as that would enable us to throw an exception if `streamIndex` is specified (not null) but no videostream has the specified index